scripts/build_awslc.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+# Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Released under Apache 2.0 license as described in the file LICENSE.
+# Author(s): Yan Peng
+
+# Remember where LNSym is
+LNSym_DIR = ${PWD}
+
+# Fetching and building AWS-LC
+git clone https://github.com/aws/aws-lc.git $HOME/aws-lc
```

Comment:
```suggestion
git clone https://github.com/aws/aws-lc.git $HOME/aws-lc --depth 1
```

Shallow clones are much faster, and we don't care about history :)

Reply:
Thanks for the suggestion!